avocado/utils/pci.py

```diff
@@ -280,6 +280,24 @@ def get_pci_id(pci_address):
         return ":".join(pci_id)
 
 
+def get_pci_info(pci_address):
+    """
+    Gets PCI info of given PCI address.
+    :param pci_address: Any segment of a PCI address (1f, 0000:00:1f, ...)
+    :return: Dictionary attribute name as key and attribute value as value.
+    :rtype: Dict
+    """
+    cmd = f"lspci -Dnvmm -s {pci_address}"
+    output = process.run(cmd, ignore_status=True, shell=True).stdout_text
+    pci_info = {}
+    if output:
+        for line in output.splitlines():
+            if line.strip():
+                s_line = line.split(":")
```

Comment:
I think above two line are not required and can be incorporate into below single line.
  ```
if output:
        for line in output.splitlines().strip():
            pci_info[s_line.split(":")[0].strip()] = (":".join(s_line.split(":")[1:])).strip()
or
if output:
        for line in output.splitlines().strip():
            line = line.split(":")
            pci_info[line[0].strip()] = (":".join(line[1:])).strip()
           
```

if condition is not required and try avoid extra variable initialization(s_line).

Reply:
Yes! I will remove s_line attribute which is not needed.

.strip() function is not supported on array. That the reason I wrote a different if condition. strip() function is available in numpy module. Let me know if we can use numpy module?